tests/acceptance/features/bootstrap/GraphApiContext.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use Psr\Http\Message\ResponseInterface;
+use TestHelpers\HttpRequestHelper;
+use TestHelpers\SetupHelper;
+use PHPUnit\Framework\Assert;
+
+require_once 'bootstrap.php';
+
+/**
+ * Context for GraphApi specific steps
+ */
+class GraphApiContext implements Context {
+
+    /**
+     * @var FeatureContext
+     */
+    private FeatureContext $featureContext;
+
+    /**
+     * @var string
+     */
+    private string $personalDriveWebDavUrl;
+
+    /**
+     * @return string
+     */
+    public function getPersonalDriveWebDavUrl(): string
+    {
+        return $this->personalDriveWebDavUrl;
+    }
+
+    /**
+     * @param string $personalDriveWebDavUrl
+     */
+    public function setPersonalDriveWebDavUrl(string $personalDriveWebDavUrl): void
+    {
+        $this->personalDriveWebDavUrl = $personalDriveWebDavUrl;
+    }
+    /**
+     * @BeforeScenario
+     *
+     * @param BeforeScenarioScope $scope
+     *
+     * @return void
+     * @throws Exception
+     */
+    public function setUpScenario(BeforeScenarioScope $scope): void
+    {
+        // Get the environment
+        $environment = $scope->getEnvironment();
+        // Get all the contexts you need in this context
+        $this->featureContext = $environment->getContext('FeatureContext');
+        SetupHelper::init(
+            $this->featureContext->getAdminUsername(),
+            $this->featureContext->getAdminPassword(),
+            $this->featureContext->getBaseUrl(),
+            $this->featureContext->getOcPath()
+        );
+    }
+
+    /**
+     * Send Graph List Drives Request
+     *
+     * @param $baseUrl
+     * @param $user
+     * @param $password
+     * @param $arguments
+     * @param string $xRequestId
+     * @param array $body
+     * @param array $headers
+     * @return ResponseInterface
+     */
+    public function listSpacesRequest(
+        $baseUrl,
+        $user,
+        $password,
+        $arguments,
+        string $xRequestId = '',
+        array $body = [],
+        array $headers = []
+    ) {
+        $fullUrl = $baseUrl;
+        if (!str_ends_with($fullUrl, '/')) {
+            $fullUrl .= '/';
+        }
+        $fullUrl .= "graph/v1.0/me/drives/" . $arguments;
+
+        return HttpRequestHelper::sendRequest($fullUrl, $xRequestId, 'GET', $user, $password, $headers, $body);
```

Comment:
you could use `HttpRequestHelper::get` if you like, similar with most other request types